app/api/routes.py

```diff
@@ -218,10 +218,15 @@ def search_results():
     page = request.args.get('page', 0, int)
     page_size = request.args.get('page_size', Config.RESOURCE_PAGINATOR.per_page, int)
 
-    search_result = index.search(f'{term}', {
-        'page': page,
-        'hitsPerPage': page_size
-    })
+    try:
```

Comment:
I wasn't catching the actual search function if it failed so I'm making it return a server error if it does.